.circleci/config.yml

```diff
@@ -309,8 +309,19 @@ jobs:
           command: |
             echo "Publishing $DOCKER_ORG/$CIRCLE_PROJECT_REPONAME:$CIRCLE_TAG"
             docker push $DOCKER_ORG/$CIRCLE_PROJECT_REPONAME:$CIRCLE_TAG
-            echo "Publishing $DOCKER_ORG/$CIRCLE_PROJECT_REPONAME:$RELEASE_TAG"
-            docker push $DOCKER_ORG/$CIRCLE_PROJECT_REPONAME:$RELEASE_TAG
+
+            case "$CIRCLE_TAG" in 
+              *-pisp*)
+                # Don't update `late5t` for an image that has a `-pisp`
```

Comment:
late5t?

Reply:
Indeed. If you have an environment variable (even if you don't care that it's public) with the same name as anything in the config file, CircleCI will change it to  ****.

So if we don't have this, during the job, it would print out "Don't update `******` for an image"... making it much harder to read.